tests/cephfs/cephfs_top/validate_fs_top_stats.py

```diff
@@ -0,0 +1,368 @@
+            "both clients"
+        )
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ip = fs_util.get_mon_node_ips()
+        mon_node_ip = ",".join(mon_node_ip)
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_1,
+            mon_node_ip,
+            extra_params=f",fs={fs_name}",
+        )
+        active_mds = fs_util.get_active_mdss(client1, fs_name)
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph tell mds.{active_mds[0]} client ls --format json"
+        )
+        client_details = json.loads(out)
+        kernel_client_ls = [item["id"] for item in client_details]
+        kernel_client_id = kernel_client_ls[0]
+        fs_util.fuse_mount(
+            [client1], fuse_mounting_dir_1, extra_params=f" --client_fs {fs_name}"
+        )
+
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"ceph tell mds.{active_mds[0]} client ls --format json"
+        )
+        client_details = json.loads(out)
+        fuse_client_ls = [item["id"] for item in client_details]
+        fuse_client_id = [
+            item for item in fuse_client_ls if item not in kernel_client_ls
+        ][0]
+        log.info(f"fuse_client_id : {fuse_client_id}")
+
+        with parallel() as p:
+            p.spawn(fs_util.run_ios, client1, fuse_mounting_dir_1, ["dd", "smallfile"])
+            p.spawn(
+                fs_util.run_ios, client1, kernel_mounting_dir_1, ["dd", "smallfile"]
+            )
+
+            log.info("Validate mount points and client ids")
+            out, rc = client1.exec_command(
+                sudo=True, cmd=f"ceph tell mds.{active_mds[0]} client ls --format json"
+            )
+            client_details = json.loads(out)
+
+            out, rc = client2.exec_command(
+                sudo=True, cmd=f"cephfs-top --dumpfs {fs_name}"
+            )
+            top_details = json.loads(out)
+            log.info(f"{top_details}")
+
+            expected_mount_root, expected_mount_point = extract_mount_info_by_id(
+                client_details, fuse_client_id
+            )
+            validate_mount_info(
+                client2,
+                fuse_client_id,
+                expected_mount_root,
+                expected_mount_point,
+                fs_name,
+            )
+            expected_mount_root, expected_mount_point = extract_mount_info_by_id(
+                client_details, kernel_client_id
+            )
+            # validate_mount_info(
+            #     client2,
+            #     kernel_client_id,
+            #     expected_mount_root,
+            #     expected_mount_point,
+            #     fs_name,
+            # )
+        log.info(
+            "scenario 3: Reboot client and validate the cephfs-top dump it should be empty"
```

Comment:
This is a negative test case, which needs to be reported if they are reset after reboots.
Can you file a bz and add that bz here and let it fail as known issue.